packages/flame/lib/src/experimental/world.dart

```diff
@@ -25,18 +26,8 @@ class World extends Component {
   bool containsLocalPoint(Vector2 point) => true;
 
   @override
-  Iterable<Component> componentsAtPoint(
-    Vector2 point, [
-    List<Vector2>? nestedPoints,
-  ]) {
-    return const Iterable.empty();
-  }
+  Vector2? localToParent(Vector2 point) => null;
```

Comment:
Isn't this an `UnimplementedException` rather than null?

Reply:
The `null` return value for the CoordinateTransform interface indicates that the given point cannot be mapped to the parent coordinate space. Which is exactly the case here: there is no mapping between the device and the world coordinates other than what is provided externally by the camera.